app/components/ui/files/publish-modal/component.js

```diff
@@ -160,67 +160,75 @@ export default Ember.Component.extend({
         xmlHttp.open("GET", 'https://cn.dataone.org/portal/token', false );
         }
         // Set the response content type
-        xmlHttp.setRequestHeader("Content-Type", "text/xml")
+        xmlHttp.setRequestHeader("Content-Type", "text/xml");
         // Let XMLHttpRequest know to use cookies
         xmlHttp.withCredentials = true;
         xmlHttp.send(null);
         
         return xmlHttp.responseText;
     },
-
-    sleep(ms) {
-        return new Promise(resolve => setTimeout(resolve, ms));
-      },
-
       
     dataoneLogin() {
         /*
         Responsible for opening the login dialog for the user. Ideally, we could
         tell when the user finishes logging in so that we know when to fetch the token
         */
-       let url = 'https://cn.dataone.org/portal/oauth?action=start&target=http://probable-cattle.nceas.ucsb.edu:4200/redirect'
+       let url = 'https://cn.dataone.org/portal/oauth?action=start&target=http://dashboard.dev.wholetale.org/redirect';
```

Comment:
These URLs should come from the `environment/config` file and be set differently for __development__ and __production__